web_monitoring/db.py

```diff
@@ -83,8 +83,8 @@ def _build_version(*, page_id, uuid, capture_time, uri, hash, source_type, title
     return version
 
 
-def _build_importable_version(*, page_url, uuid, capture_time, uri, hash,
-                             source_type, title, source_metadata=None):
+def _build_importable_version(*, page_url, uuid=None, capture_time, uri, version_hash,
+                             source_type, page_title, site_agency, site_name, source_metadata=None):
```

Comment:
This signature change (and the associated implementation changes) are probably not quite right, but they were more expedient than changing where we call `_build_importable_version(**version_from_ia_module)`. The params we expect here are not really compatible with what the `internetarchive` module spits out, so we need to figure out what to change where for that.

Reply:
Until some point in the middle of my work on `Client`, I lost track of the fact that bulk-import mashes attributes in with version attributes. I still find this confusing and potentially bad: who checks that the site_name is consistent? But agreed that this fix is needed at least for now and that my original was just a mistake.